src/ClientModel/SamplingDataPostProcessor.cpp

```diff
@@ -141,54 +142,56 @@ PostProcessedSamplingData SamplingDataPostProcessor::ProcessSamples(
     const CallstackData& callstack_data, const CaptureData& capture_data,
     const ModuleManager& module_manager, bool generate_summary) {
   // Unique call stacks and per thread data
-  callstack_data.ForEachCallstackEvent(
```

Comment:
What happened here? Is it properly formatted?

Reply:
I think that `thread_sample_data->sampled_callstack_id_to_events[event.callstack_id()].emplace_back(event);` is now the longest line so `clang-format` tries to keep it a single line by indenting differently. With lambdas the result of `clang-format` is always hard to predict.

Reply:
Ah makes sense. Thanks